core/services/relay/evm/chain_reader.go

```diff
@@ -262,7 +274,7 @@ func (cr *chainReader) getEventInput(def types.ChainReaderDefinition, contractNa
 }
 
 func verifyEventInputsUsed(chainReaderDefinition types.ChainReaderDefinition, indexArgNames map[string]bool) error {
```

Comment:
Is there a reason why this is called `verifyEventInputsUsed` rather than something more specific like `verifyEventInputsIndexed`? Seems hard to infer what the first one means without looking at the code.